careless_stats/careless_stats.md

```diff
@@ -0,0 +1,149 @@
+# Using `careless.stats`
+
+This notebook describes all the machine learning and data crossvalidation metrics that `careless` can compute. As an example, we describe the metrics from the analysis of anomalous signal from a thermolysin serial femtosecond crystallography dataset, as done in [Hekstra, Wang et al. 2025](https://www.biorxiv.org/content/10.1101/2024.07.22.604476v1). This notebook heavily draws on explanations from [Aldama, Dalton, and Hekstra 2023](https://journals.iucr.org/d/issues/2023/09/00/qi5002/index.html), which is good prerequisite reading for this notebook; this notebook also breaks down the common validation scripts found in `dw-examples`. 
+
+# Table of Contents
+1. [Elementary crystallographic statistics](#Elementarycrystallographicstatistics)
+2. [Validation statistics](#Validationstatistics)
+3. [Machine learning metrics](#Machinelearningmetrics)
+
+
+## Elementary crystallographic statistics <a name="Elementarycrystallographicstatistics"></a>
+
+### Completeness and multiplicity
+`careless.stats` also computes standard crystallography statistics, such as completeness and $I/\sigma(I)$. 
+
+```
+>careless.completeness ./thermolysin_dw/thl_1p8A_grid_both.mtz -o ./thermolysin_dw/completeness.csv -b 10
```

Comment:
angle brackets should not be used to indicate the command promp, because they preclude copying and pasting the command.